packages/editor/src/components/post-title/index.js

```diff
@@ -221,15 +222,33 @@ function PostTitle( _, forwardedRef ) {
 				};
 			} );
 		},
-		__unstableDisableFormats: true,
+		__unstableDisableFormats: false,
 		preserveWhiteSpace: true,
 	} );
 
+	const richEditorRef = useMergeRefs( [ richTextRef, ref ] );
```

Comment:
You're passing the rich text ref to plain text

```suggestion
	const richEditorRef = useMergeRefs( [ rawText ? null : richTextRef, ref ] );
```

Reply:
Thank you 👍

Reply:
Maybe it's better to split this in two components so it's less confusing

Reply:
Yes. I think that could be useful.

Reply:
Now internally we have two separate components.

I didn't expose these two components in the package as separate files.

Why? Because they share much of the same logic (especially around updating and focus management) and it didn't seem worth taking the refactor too far (at least not in this PR). 

If we want to create two dedicated components in future and remove the `rawText` prop then we could do that. Perhaps marking the prop as `unstable` would be worthwhile for now?